config/config.go

```diff
@@ -31,6 +31,21 @@ type MessageProtocolConfig struct {
 	Settings      map[string]interface{} `mapstructure:"settings" json:"settings"`
 }
 
+type ManualWarpMessage struct {
+	UnsignedMessageBytes    string `mapstructure:"unsigned-message-bytes" json:"unsigned-message-bytes"`
```

Comment:
Follow up on the PR description, could we use Warp message ID in the config and use `GetMessage()`?

Reply:
Looks like this endpoint is available in avalanchego v1.11.0-fuji, so we should be good to update this PR to match. I'm also considering merging this PR as is and following up with another PR that makes this change while also updating to the latest avalanchego.

Reply:
I'm good with that since other PRs are dependent on this one as base, can you create a new issue for it and link it here?

Reply:
Sure thing: https://github.com/ava-labs/awm-relayer/issues/171